### DISCUSSION TITLE
External validators

### DISCUSSION THREAD

**Discussion by johnnymatthews** [entropyxyz/community#114] [General]
Entropy is in the early stages of the testnet, and the team has yet to announce a specific date for accepting external validator nodes on the network. It's crucial to note that the performance of the testnet will play a significant role in determining how and when external validators will be accepted onto the network. The team is diligently monitoring the testnet's performance to ensure a smooth and secure transition for external validators in the future.

Testnet performance over the next few months will strongly indicate when and how we will manage external validators on the network. We'll make an announcement closer to the time regarding how you can become a validator.

**Comment by aditya-manit** [comment]
Hi, was wondering if there is any update on this now ?

**Reply by johnnymatthews** [reply]
There are no updates as of yet. We'll be sure to make an annoucement when there's news.
